websites/P/Padlet/presence.ts

```diff
@@ -0,0 +1,90 @@
+const presence = new Presence({
+  clientId: "760981805695762441"
+});
+
+// Variables 
+presence.on("UpdateData", async () => {
+  // Presence Data
+  const timestamp = presence.getTimestamps(Date.now(), 0),
```

Comment:
Okay literally, why. Use this and don't complicate ``Math.floor(Date.now() / 1000)``.